add-on/chrome/bootstrap.js

```diff
@@ -671,10 +683,17 @@ var WindowListener = {
           null,
           box.PRIORITY_WARNING_LOW,
           [{
-            label: initStrings.label,
+            label: this._roomSessionConnected ? initStrings.label : pendingBtnString,
             accessKey: initStrings.accessKey,
             isDefault: false,
             callback: (event, buttonInfo, buttonNode) => {
+              // Prevent any action before connecting to a room.
+              if (!this._roomSessionConnected) {
+                // You must throw an error to prevent the notification bar
+                // from closing when the button is clicked.
+                throw new Error('Prevent notification bar close');
+              }
```

Comment:
We should get ui feedback before going too far here. I would think we would want to change the message (not button) text and perhaps not bother showing the "pause" button.